src/components/editMeasure/testCases/components/testCaseLanding/common/ActionCenter/ActionCenter.test.tsx

```diff
@@ -468,5 +468,232 @@ describe("ActionCenter Component", () => {
       );
       expect(screen.getByTestId("delete-action-btn")).toBeDisabled();
     });
+
+    describe("Make JSON Match UI Button", () => {
+      beforeEach(() => {
+        (useFeatureFlags as jest.Mock).mockReturnValue({
+          MakeJSONMatchUI: true,
+        });
+      });
+
+      it("should not display Make JSON Match UI button when feature flag is off", () => {
+        (useFeatureFlags as jest.Mock).mockReturnValue({
+          MakeJSONMatchUI: false,
+        });
+
+        render(
+          <MemoryRouter>
+            <ActionCenter
+              selectedTestCases={[
+                { id: "1", validResource: true, title: "Test Case 1" },
+              ]}
+              canEdit={true}
+              isQDM={false}
+              isDraft={true}
+            />
+          </MemoryRouter>
+        );
+
+        expect(
+          screen.queryByTestId("make-json-match-ui-action-btn")
+        ).not.toBeInTheDocument();
+      });
+
+      it("should not display Make JSON Match UI button when user does not have edit access", () => {
+        render(
+          <MemoryRouter>
+            <ActionCenter
+              selectedTestCases={[
+                { id: "1", validResource: true, title: "Test Case 1" },
+              ]}
+              canEdit={false}
+              isQDM={false}
+              isDraft={true}
+            />
+          </MemoryRouter>
+        );
+
+        expect(
+          screen.queryByTestId("make-json-match-ui-action-btn")
+        ).not.toBeInTheDocument();
+      });
+
+      it("should not display Make JSON Match UI button for QDM measures", () => {
+        render(
+          <MemoryRouter>
+            <ActionCenter
+              selectedTestCases={[
+                { id: "1", validResource: true, title: "Test Case 1" },
+              ]}
+              canEdit={true}
+              isQDM={true}
+              isDraft={true}
+            />
+          </MemoryRouter>
+        );
+
+        expect(
+          screen.queryByTestId("make-json-match-ui-action-btn")
+        ).not.toBeInTheDocument();
+      });
+
+      it("should display Make JSON Match UI button for QI-Core measures when feature flag is enabled and user has edit access", () => {
```

Comment:
this testcase and the following test case(`should enable Make JSON Match UI button when one test case is selected`) are doing the same thing. we can just keep one and update the name, reducing the build time.